### ISSUE TITLE
Different icon name for Caja computer

### ISSUE THREAD

**Issue by bil-elmoussaoui** [mate-desktop/caja#721]
The current icon name of Caja computer is https://github.com/mate-desktop/caja/blob/master/data/caja-computer.desktop.in.in#L6 `computer` which is not an application icon..
https://specifications.freedesktop.org/icon-naming-spec/icon-naming-spec-latest.html
Can you please rename it to something else?

**Comment by kn00tcn** [comment]
you want it to be 'system-file-manager'?

**Comment by bil-elmoussaoui** [comment]
@kn00tcn Yeah, whatever as soon as it"s an application icon, because this break some themes (device icon used instead of an application one which might look ugly..)

**Comment by tamplan** [comment]
Hi,

According to me, i think the name of the icon is correct, a computer is a device not an application if i understand right the link to give about specification, so the name follow the specifications you speak about.

Regards.
